### PR TITLE
Update colorbutton module. Remove patches.

### PR DESCRIPTION
**Pull request by ddrozdik**
I have got access to the module and added patches and released a new version. here is an update for OpenY

Steps for review:
- [ ] Review code changes

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1879 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1879 |
| Installation process | http://install.openy-dev.ffwua.com/build1879/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1879/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1879

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1879/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1879/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1879/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1879/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1879/blogWCAG2AAhtmlcs.txt

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1879/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1886 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1886 |
| Installation process | http://install.openy-dev.ffwua.com/build1886/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1886/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build1886

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1886/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1886/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1886/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1886/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1886/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1886/profiles/contrib/openy/build/reports/behat

**Review by david-hernandez — APPROVED** [review]
Looks fine. I tested the composer update and get the new version of the module.

File: drupal-org.make [hunk 1]
```diff
@@ -67,10 +67,7 @@ projects[panelbutton][subdir] = contrib
 projects[panelbutton][version] = 1.0
 projects[panelbutton][patch][] = "https://www.drupal.org/files/issues/panelbutton-libraries-2881820.patch"
 projects[panelbutton][patch][] = "https://www.drupal.org/files/issues/panelbutton-hook-requirements-2881818_0.patch"
-projects[colorbutton][subdir] = contrib
-projects[colorbutton][version] = 1.0
-projects[colorbutton][patch][] = "https://www.drupal.org/files/issues/colorbutton-libraries-2881822-3.patch"
-projects[colorbutton][patch][] = "https://www.drupal.org/files/issues/colorbutton-hook-requirements-2881819_0.patch"
+projects[colorbutton] = 1.0
```
Comment (alexschedrov) [inline drupal-org.make:70]: Should be 1.1.

Comment (ddrozdik) [reply drupal-org.make:70]: yeah, correct. I will fix

**Pull request merged**